lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/files/JobJarMaker.java

```diff
@@ -93,8 +93,7 @@ private static File toFile(final FileResource fileProto) {
   }
 
   public File createJobSubmissionJAR(
-      final JobSubmissionEvent jobSubmissionEvent,
-      final Configuration driverConfiguration) throws IOException {
+      final Configuration configuration, Set<FileResource> globalFileSet, Set<FileResource> localFileSet, String configurationFileName) throws IOException {
```

Comment:
Why are we removing the JobSubmissionEvent abstraction instead of just using "addLocalFile" and "addGlobalFile" to build a new event? I'm concerned that this is moving in the wrong direction.

Reply:
The JobJarMaker was previously only designed for JobSubmissionEvent. The work here I am going to assemble the Evaluator Jar in Driver, which is doing exact the same logic inside this function. So I refactored the function. Maybe I should rename it to avoid confusion?

Reply:
I will refactor to create a new method.

Reply:
refactored code is uploaded. I kept the previous interface as it was.